pkg/process/util/containers.go

```diff
@@ -28,6 +30,10 @@ func SetContainerSources(names []string) {
 // GetContainers autodetects the best backend from available sources
 // if the users don't specify the preferred container sources
 func GetContainers() ([]*containers.Container, error) {
+	if runtime.GOOS == "darwin" {
+		return nil, fmt.Errorf("containers collection not available for darwin")
```

Comment:
Will this not print an error every 10s in the process check?

Reply:
For `process` and `process_rt` checks, we don't verify the error returned by `GetContainers()`

https://github.com/DataDog/datadog-agent/blob/master/pkg/process/checks/process.go#L82

https://github.com/DataDog/datadog-agent/blob/master/pkg/process/checks/process_rt.go#L57